app/models/change_note.rb

```
@@ -25,17 +31,20 @@ def build
 
   def create_from_top_level_change_note
     return unless change_note
-    ChangeNote.create!(
-      content_item: content_item,
-      note: change_note,
-      public_timestamp: Time.zone.now
-    )
+    ChangeNote.
+      find_or_create_by(content_item: content_item).
+      update(
```

Would it be better to raise here, might silently fail